content/site/about-us.md

```diff
@@ -15,4 +15,21 @@ We know this is a lofty goal, though, and we don't want to do it alone. If you'r
  
 If you’d like to get in touch with us, [our Discord](https://discord.gg/FMcvc6T) is probably the best way to do so.
 
+## Sponsors
+
+<a href="https://www.thepolyglotdeveloper.com/" target="_blank" rel="noopener noreferrer sponsored"><img alt="The Polyglot Developer" src="/sponsors/the-polyglot-developer.svg" width="300"/></a>
+
+## Statement of Ethics {#ethics}
+
+We never want to end up in a place where our educational content or the community or
+experience around that content is compromised. Not by money and not by any potential
+of harmful members within said community. As a result,
+[we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Maybe update this to match the statement in the README? I'm not sure which is newer.

Reply:
Oh right. Thank you